src/MemoryUsage.h

```diff
@@ -114,29 +114,99 @@ extern uint8_t *__heap_end;
 extern uint8_t *__bss_start;
 extern uint8_t *__bss_end;
 
+extern int numStackComputeCalls;
+extern int mu_stack_size;
+
 //
 // Memory addresses
 //
+namespace MU {

```

Comment:
New namespace "MU".  It may not really be necessary, but I thought the library might be heading in that direction with `mu_StackCount()` and `mu_stack_size`.  I left those alone to ensure these changes are backwards-compatible.